content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -30,14 +30,14 @@ instead, which is a stable feature.
     $ kubectl apply -f https://preliminary.istio.io/operator.yaml
     {{< /text >}}
 
-1. (Optionally) change profiles from the default profile to one of the following profiles:
+1. (Optionally) change profiles from the demo profile to one of the following profiles:
 
 {{< tabset cookie-name="profile" >}}
 
-{{< tab name="permissive mutual TLS" cookie-value="permissive" >}}
+{{< tab name="demo" cookie-value="permissive" >}}
 When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
 mutual TLS traffic. Clients send plaintext traffic unless configured for
-[mutual TLS migration](/docs/tasks/security/mtls-migration/).
+[mutual TLS migration](/docs/tasks/security/mtls-migration/). This profile is installed during the first step.
```

Comment:
If it's already installed in the first step, why is it an option here?

Reply:
in case user wants to switch back to demo profile :) if you think it is cleaner, i can remove this demo tab, along with demo-auth tab (since that is gone too).

Reply:
Why shouldn't users be able to use the demo profile with strict auth enabled?

Reply:
@frankbu there are two profiles.  demo, and demo-auth.  When we spoke via slack, I had the intention of unifying the documention post 1.4 with the profiles we have, so people can easily switch profiles to execute different tasks.  This would replace the helm operations.

Reply:
Something is not very clear here. Let me try to understand.

Step 1: User applies operator.yaml, which installs the demo profile.
Step 2 (optional): User applies one of the operator-profile-xxx.yaml files to install a different profile.

Question: when a user wants to install something different from the demo profile, why are two steps needed? Why can't the user just apply operator-profile-xxx.yaml, i.e., just one step?

Are the operator-profile-xxx.yaml files additive to the operator.yaml? If so, then how can operator.yaml be starting with the demo profile ... things would have to be removed (not added) to change to a different profile.

What am I missing here?

Reply:
OK, I looked at the yaml files and I understand what's going on now :-)

I think we need to add something in the doc to explain that operator.yaml is starting the operator and by default is configuring a control plane for profile: demo. Then the second step is just changing the profile name (operator then knows what to do).

This would be cleaner if there was a way to pass in the profile into the first apply command, selector label maybe? Then we wouldn't need two steps.

Reply:
looking at the yamls was good step :)

selector sounds reasonable, although we are a little crunched for time working on the operator to fine tune how the deployment works.  What this would allow us to do though is bundle all of . the profiles into operator.yaml and select the one we want from one operator.yaml IIUC.